### PR TITLE
Release 6.12.3

### PR DESCRIPTION
**Pull request by atugushev**

### CODE REVIEW

**Review by hramezani — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 6.12.3 (2022-02-24)
```
Comment (atugushev) [inline CHANGELOG.md:1]:
```suggestion
## 6.12.3 (2022-03-01)
```

**Comment by atugushev** [comment]
Thanks @hramezani!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+## 6.12.3 (2022-03-01)
```
Comment (richafrank) [inline CHANGELOG.md:1]: Need to update the year!
```suggestion
## 6.12.3 (2023-03-01)
```

Comment (atugushev) [reply CHANGELOG.md:1]: Thanks!

Comment (atugushev) [reply CHANGELOG.md:1]: Addressed in https://github.com/jazzband/pip-tools/pull/1817

**Pull request merged**